src/launcher/features/settings/Cards/Authentication.tsx

```diff
@@ -61,17 +61,13 @@ export default () => {
                 {token == null ? (
                     <Col className="small text-muted">
                         To access restricted app sources from Nordic
-                        Semiconductor, get an identity token from{' '}
+                        Semiconductor, you need an identity token. Generating a
+                        token is described in{' '}
                         <ExternalLink
-                            href="https://files.nordicsemi.com/ui/user_profile"
-                            label="Nordic Semiconductor JFrog portal"
+                            href="https://docs.nordicsemi.com/bundle/nrf-connect-desktop/page/working_with_authentications_tokens.html#generating-a-new-token"
+                            label="the nRF Connect for Desktop documentation"
                         />
-                        .{/* if we have the docs published: */}
-                        {/* To access restricted app sources from Nordic */}
-                        {/* Semiconductor, get an identity token from the{' '} */}
-                        {/* <ExternalLink href="https://files.nordicsemi.com/ui/user_profile" label="Nordic Semiconductor JFrog portal" /> */}
-                        {/* . Read <ExternalLink href="https://docs.nordicsemi.com/bundle/nrf-connect-desktop/page/working_with_authentications_tokens.html" label="Working with identity tokens"/> for */}
-                        {/* detailed steps. */}
```

Comment:
I'd prefer this previously approved version. Though we provide 2 links instead of 1 it will be quicker to navigate directly to the JFrog portal from the app, skipping docs step when not required.
Feel free to reject if completely disagree

Reply:
When people know about this, they need neither the docs not the portal link. 

My understanding from Greg's previous comments was that we replace the portal link with the docs link. But we can also ask him when he is back what he prefers. 

The only place where I really would like to have only a single link is in the Settings pane, because I would like to keep that concise.